pkg/api/pluginproxy/ds_proxy.go

```diff
@@ -18,6 +18,7 @@ import (
 	"github.com/grafana/grafana/pkg/infra/httpclient"
 	glog "github.com/grafana/grafana/pkg/infra/log"
 	"github.com/grafana/grafana/pkg/infra/tracing"
+	"github.com/grafana/grafana/pkg/middleware/loggermw"
```

Comment:
Am I allowed to have a dependency on this package here? Or should I extract the SanitizeURL function to some other place and reference it from both the `loggermw` and `ds_proxy`?

Reply:
This should be fine for now, though it might be better off moving `SanitizeURL` to `pkg/infra/log` perhaps? WDYT @marefr since has been involved more in this refactor before

Reply:
It doesn't really have a relation with log besides the resulting value being logged so not sure about that package, but somewhere else would be good. Possibly in pkg/util?

Noticed it's used here as well https://github.com/grafana/grafana-enterprise/blob/7cc84ed58504b7f9c56d62cbf10860c1ebad7d65/src/pkg/extensions/auditing/middleware.go#L456

Reply:
It is moved, I guess I just need to open a PR in enterprise as well?